docs/releases/g_1_1.md

```diff
@@ -0,0 +1,17 @@
+# iRobot® Create® 3 Release G.1.1
+
+This is the first production Create® 3 release.
```

Comment:
Should we have a link to download the release?

Reply:
Maybe for now we ought to have a link at the top of the page that says "the most recent release can always be found at http://edu.irobot.com/create3-latest-fw" and once we get the versioned FW API up, we can update this page again.